org.eclipse.mylyn.commons.context/src/org/eclipse/mylyn/common/context/CommonContextPlugin.java

```diff
@@ -0,0 +1,53 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Frank Becker and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Frank Becker - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.mylyn.common.context;
+
+import org.eclipse.mylyn.monitor.core.InteractionEvent;
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+
+public class CommonContextPlugin implements BundleActivator {
+
+	public static final String ID_PLUGIN = "org.eclipse.mylyn.common.context"; //$NON-NLS-1$
+
+	private static CommonContextPlugin INSTANCE;
+
+	private ContextCallBack contextCallBack;
```

Comment:
Please do not introduce this mutability, we will have a lot of troubles with it:
* who and then should set it
* how to make sure that it is _already_ set before we need it
* and so on